db/migrate/20210103105959_rename_services_to_categories_and_make_them_enum.rb

```diff
@@ -0,0 +1,27 @@
+class RenameServicesToCategoriesAndMakeThemEnum < ActiveRecord::Migration[6.1]
+  Ad = Class.new(ActiveRecord::Base)
+
+  CATEGORIES = {
+    "Smještaj" => 0,
+    "Prijevoz" => 1,
+    "Usluga popravka" => 2,
+    "Medicinska pomoć" => 3,
+    "Ostalo" => 4
+  }
+
+  def up
+    add_column :ads, :category, :integer, null: false, default: 0
+    Ad.all.each do |ad|
```

Comment:
It's better to use find_each if it's a bigger dataset, since it loads the objects in batches

Reply:
Fixed in https://github.com/vlado/earthquake-croatia/pull/96/commits/53d9d1d0dd10fd622c8c91cfbcab31650398ab1e